docs/book/src/cronjob-tutorial/testdata/project/config/default/webhookcainjection_patch.yaml

```diff
@@ -1,13 +1,13 @@
 # This patch add annotation to admission webhook config and
```

Comment:
we have the same in the docs/book/src/multiversion-tutorial/testdata/project/config/default/